<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bookings_screen</class>
 <widget class="QMainWindow" name="bookings_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bookings</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="bookings_table">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>711</width>
      <height>371</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Customer Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Single/Monthly</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Horse Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Private Horse</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type of event</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="home_button">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_booking_button">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Booking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_booking_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>221</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Booking</string>
    </property>
   </widget>
   <widget class="QPushButton" name="toggle_lang_button">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>0</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Toggle English/German</string>
    </property>
   </widget>
   <widget class="QPushButton" name="edit_booking_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Booking</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
